Framework/Kernel/src/Material.cpp

```diff
@@ -180,6 +180,11 @@ void Material::calculateTotalScatterXSection() {
     m_totalScatterXSection = weightedTotal;
   }
 }
+
+void Material::setAttenuationProfile(AttenuationProfile attenuationOverride) {
+  m_attenuationOverride = attenuationOverride;
```

Comment:
To espouse good practice can we do `m_attenuationOverride = std::move(attenuationOverride);`